presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -193,6 +193,8 @@
 
     private Duration partitionLeaseDuration = new Duration(0, TimeUnit.SECONDS);
 
+    private Long missingPartitionPredicateThreshold = 1L;
```

Comment:
We don't need this to long, and lets have some good defaults. 100 may be?

Reply:
👍🏼 Need to fix this and the unit test. That's the reason I have the PR in draft phase. Was hoping to get feedback on the 
- counting approach 
- name of config